resources/definitions/dagoma_neva_magis.def.json

```diff
@@ -0,0 +1,78 @@
+{
+    "id": "dagoma_neva_magis",
+    "name": "Dagoma NEVA Magis",
+    "version": 2,
+    "inherits": "fdmprinter",
+    "metadata": {
+        "visible": true,
+        "author": "Dagoma",
+        "manufacturer": "Dagoma",
+        "file_formats": "text/x-gcode",
+        "platform": "neva.stl",
+        "platform_offset": [ 0, 0, 0]
+    },
+    "overrides": {
+        "machine_width": {
+            "default_value": 195.55
+        },
+        "machine_height": {
+            "default_value": 205
+        },
+        "machine_depth": {
+            "default_value": 195.55
+        },
+        "machine_center_is_zero": {
+            "default_value": true
+        },
+        "machine_head_with_fans_polygon": {
+            "default_value": [
+                [-36, -42],
+                [-36, 42],
+                [36, 42],
+                [36, -42]
+            ]
+        },
+        "gantry_height": {
+          "default_value": 0
+        },
+        "machine_shape": {
+            "default_value": "elliptic"
+        },
+        "machine_gcode_flavor": {
+            "default_value": "RepRap"
+        },
+        "machine_start_gcode": {
+            "default_value": ";Gcode by Cura\nG90\nG28\nM107\nM109 R100\nG29\nM109 S{material_print_temperature_layer_0} U-55 X55 V-85 Y-85 W0.26 Z0.26\nM82\nG92 E0\nG1 F200 E6\nG92 E0\nG1 F200 E-3.5\nG0 Z0.15\nG0 X10\nG0 Z3\nG1 F6000\n"
+        },
+        "machine_end_gcode": {
+            "default_value": "\nM104 S0\nM106 S255\nM140 S0\nG91\nG1 E-1 F300\nG1 Z+3 E-2 F9000\nG90\nG28\n"
+        },
+        "material_diameter": {
```

Comment:
`material_diameter` is now in the extruder definitions.

Reply:
Fixed in ed1e47d